ios/Nodes/REAFormatNode.m

```diff
@@ -0,0 +1,34 @@
+#import "REAFormatNode.h"
+#import "REAValueNode.h"
+#import "REANodesManager.h"
+
+@implementation REAFormatNode {
+  NSNumber *_value;
+  NSString *_format;
+}
+
+- (instancetype)initWithID:(REANodeID)nodeID config:(NSDictionary<NSString *,id> *)config
+{
+  if ((self = [super initWithID:nodeID config:config])) {
+    _value = config[@"value"];
+    _format = config[@"format"];
```

Comment:
we can perhaps precompute `NSNumberFormatter` here already instead of allocating it each time `evaluate` gets called